common/lib/xmodule/xmodule/modulestore/split_mongo/caching_descriptor_system.py

```diff
@@ -131,6 +131,13 @@ def _load_item(self, usage_key, course_entry_override=None, **kwargs):
 
         class_ = self.load_block_type(block_data.block_type)
         block = self.xblock_from_json(class_, course_key, block_key, block_data, course_entry_override, **kwargs)
+
+        # TODO Once PLAT-1055 is implemented, we can expose the course version
+        # information within the key identifier of the block.  Until then, set
+        # the course_version as a field on the returned block so higher layers
+        # can use it when needed.
+        block.course_version = version_guid
```

Comment:
For instance, course creation seems to retrieve and update blocks in the modulestore without looking at the cache (unsurprisingly):

https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/modulestore/split_mongo/split.py#L1967
->
https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/modulestore/split_mongo/split.py#L1065
->
https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/modulestore/split_mongo/split.py#L1055
->
https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/modulestore/split_mongo/split.py#L789